content/rs/administering/designing-production/supported-platforms.md

```diff
@@ -17,7 +17,7 @@ systems, cloud environments, and virtual environments.
 | RHEL/CentOS 7 | 7.0, 7.1, 7.2, 7.3, 7.4, 7.5, 7.6<br>Requires at least "Minimal Install" configuration. For additional considerations, refer to [CentOS / RHEL 7 firewall configuration]({{< relref "/rs/installing-upgrading/configuring/centos-rhel-7-firewall.md" >}}). |
 | Oracle Linux | 6.7, 6.8, 6.9; 7.0, 7.1, 7.2, 7.3, 7.4, 7.5 |
 | Amazon Linux | All 64-bit Versions |
-| Docker | At this time, official Redis Enterprise Software Docker images are Linux-based images and therefore are only certified for production use on a Linux host. All other OS hosts are certified for Development and Testing only. |
+| Docker | At this time, official Redis Enterprise Software Docker images are Linux-based images and are certified for Development and Testing only. |
```

Comment:
I would say something like:
"Redis Enterprise Software Docker images are certified for Development and Testing only. "